ConsoleApp/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace ConsoleApp
+{
+    using System;
+    class ConsoleApp
+    {
+        static void ShowValue(int value)
+        {
+            Console.Write("+" + (value < 0 ? "(" + value + ")" : value));
+        }
+        static long SumCalculate(int x, int y)
+        {
+            long sum = 0;
+            Console.Write("Sum = ");
+            if (x < y)
+            {
+                Console.Write("{0}", x);
```

Comment:
lines 26-32 copypasted, it's better to replace X and Y if X > Y and have a single code